forge-gui/res/cardsfolder/upcoming/tyranid_invasion.txt

```diff
@@ -0,0 +1,7 @@
+Name:Tyranid Invasion
+ManaCost:3 G
+Types:Sorcery
+A:SP$ Token | TokenOwner$ You | TokenAmount$ X | TokenScript$ g_3_3_tyranid_warrior
+SVar:X:PlayerCountOpponents$Amount
+DeckHas:Ability$Token
+Oracle:Create a number of 3/3 green Tyranid Warrior creature tokens with trample equal to the number of opponents you have.
```

Comment:
AI:RemoveDeck:Random – only good in multiplayer, we don't have a better AI hint yet